packages/shared/src/styles/base.css

```diff
@@ -215,14 +215,11 @@ body {
   --theme-shadow-shadow2: color-mix(in srgb, theme('colors.raw.pepper.90'), transparent 60%);
   --theme-shadow-shadow3: color-mix(in srgb, theme('colors.raw.pepper.90'), transparent 36%);
 
-
-
-
-
-
-
-
-
+  /* Blur */
+  --theme-blur-blur-highlight: color-mix(in srgb, theme('colors.raw.pepper.70'), transparent 12%);
+  --theme-blur-blur-baseline: color-mix(in srgb, theme('colors.raw.pepper.90'), transparent 12%);
+  --theme-blur-blur-bg: color-mix(in srgb, theme('colors.raw.pepper.90'), transparent 36%);
+  --theme-blur-blur-glass: color-mix(in srgb, theme('colors.raw.salt.90'), transparent 92%);
```

Comment:
Were these blur theme colors new? or just not added here yet

cc. @rebelchris

Reply:
I'm not sure if they are new from the design point of view, but they were definitely missing from this definition.

Reply:
It's ok as Chris mentioned we can merge the PR with these as he can take a look later

Reply:
Based on [latest comment from Chris](https://dailydotdev.slack.com/archives/C06CUKX1P3N/p1711464791569759?thread_ts=1711460922.327719&cid=C06CUKX1P3N), we can merge this and figure it out later.